README.md

```diff
@@ -1,8 +1,3 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
 
```

Comment:
- [ ] Clicking on the following link in the screenshot below should take the user to the `Run Tests` part of the getting started section, kindly fix this as broken links make your readme file look unprofessional.
![image](https://github.com/waelelsafty07/vet/assets/81932478/1359b926-afb0-4193-b3db-729460314356)

- [ ] Also kindly remove commented lines from your reademe file for a a cleaner more professional file.
![image](https://github.com/waelelsafty07/vet/assets/81932478/2b8aca17-dadf-4676-9c88-8f4494825f37)
![image](https://github.com/waelelsafty07/vet/assets/81932478/4b90a195-3732-42b0-bed2-07e9afc14c0f)